<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SankarRama_gift" failures="1" tests="3" name="Test" time="2.628" errors="0" timestamp="2025-03-22T15:47:05 EDT">
  <testcase classname="TestNGPractice.TestClass1" name="TestMethod2" time="0.011"/>
  <testcase classname="TestNGPractice.TestClass1" name="loginToSalesforce" time="2.573">
    <failure type="java.lang.AssertionError" message="expected [Lightning Experience] but found [Login | Salesforce]">
      <![CDATA[java.lang.AssertionError: expected [Lightning Experience] but found [Login | Salesforce]
at TestNGPractice.TestClass1.loginToSalesforce(TestClass1.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesforce -->
  <testcase classname="TestNGPractice.TestClass1" name="testMethod1" time="0.001"/>
</testsuite> <!-- Test -->
